index.bs

```diff
@@ -3,13 +3,14 @@ Title: Multi-Screen Window Placement
 Shortname: multi-screen
 Abstract: This document defines a web platform API that allows script to query the device for information about its screens, and place content on specific screens.
 Status: ED
-URL: https://w3c.github.io/window-placement
-Level: 1
-Editor: Joshua Bell, Google Inc. https://google.com, jsbell@google.com
-Editor: Mike Wasserman, Google Inc. https://google.com, msw@google.com
+TR: https://www.w3.org/TR/window-placement/
+URL: https://w3c.github.io/window-placement/
+Level: None
+Editor: Joshua Bell, Google Inc. https://google.com, jsbell@google.com, w3cid 61302
+Editor: Mike Wasserman, Google Inc. https://google.com, msw@google.com, w3cid 116970
 Repository: w3c/window-placement
 Group: secondscreenwg
-Test Suite: https://github.com/web-platform-tests/wpt/tree/master/screen-details
+Test Suite: https://github.com/web-platform-tests/wpt/tree/master/window-placement
```

Comment:
Can we list multiple wpt dirs here? Both screen-details and window-placement are pertinent; no worries if not.

Reply:
Not directly as Bikeshed expects to see only one URL there.
However, the setting can be turned into a "custom key" with a `!` prefix as in:

```suggestion
!Test Suite: <a href="https://github.com/web-platform-tests/wpt/tree/master/window-placement">https://github.com/web-platform-tests/wpt/tree/master/window-placement</a>
!Test Suite: <a href="https://github.com/web-platform-tests/wpt/tree/master/screen-details">https://github.com/web-platform-tests/wpt/tree/master/screen-details</a>
```

Reply:
sgtm if that's an acceptable way to cite both wpt dirs. I'll attempt to commit your suggestion and merge.